### PR TITLE
UT3 Hellbender Rear Gun Gunner Pose & Cam

### PR DESCRIPTION
**Pull request by HellDragon-HK**
I did some work getting this pose, hope you like it, once again the arms just refuse to do what I wanted so I got it as best I could, further attempts to improve it did the opposite....also I'm trying to make these positions work with 2 different driving animations, the stock and then my preferred skaarj hybrid model which unfortunately has the legs bent lower (by a lot) and the arms in a different position, the good thing about this animation is it actually fits the stock UT vehicles perfectly the bad it's hard to fit into these UT3 seats without messing up the stock model's pose.

Cams slightly adjusted as well

### CODE REVIEW

File: Classes/UT3HellbenderRearGunPawn.uc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright © 2008, 2014 GreatEmerald
+ * Copyright © 2017 HellDragon (still don't feel right about it but here you go)
```
Comment (GreatEmerald) [inline Classes/UT3HellbenderRearGunPawn.uc]: Don't put comments into the copyright line.

File: Classes/UT3HellbenderRearGunPawn.uc [hunk 2]
```diff
@@ -39,10 +40,96 @@
 
 class UT3HellbenderRearGunPawn extends ONSPRVRearGunPawn;
 
+Var rotator SpineDrive;
+Var rotator ThighDriveL,ThighDriveR;
+Var rotator CalfDriveL,CalfDriveR;
+Var rotator FootDriveL,FootDriveR;
+Var rotator ArmDriveL,ArmDriveR;
+Var rotator ForeArmDriveL, ForeArmDriveR;
+Var rotator NeckDrive;
```
Comment (GreatEmerald) [inline Classes/UT3HellbenderRearGunPawn.uc]: Same comment here: either make these local variables inside the `AttachDriver` function, or...

File: Classes/UT3HellbenderRearGunPawn.uc [hunk 3]
```diff
@@ -39,10 +40,96 @@
...
+var bool bAttachedDriver;
+
+simulated function AttachDriver(Pawn P)
+{
+    super.AttachDriver(P);
+    bAttachedDriver=true;
+
+    NeckDrive.Yaw=4000;
```
Comment (GreatEmerald) [inline Classes/UT3HellbenderRearGunPawn.uc]: ...remove these definitions and put them into `defaultproperties` instead.

File: Classes/UT3HellbenderRearGunPawn.uc [hunk 4]
```diff
@@ -39,10 +40,96 @@
...
+    P.SetBoneRotation('Bip01 L ForeArm');
+    P.SetBoneRotation('Bip01 R ForeArm');
+    P.SetBoneRotation('Bip01 L Thigh');
+    P.SetBoneRotation('Bip01 R Thigh');
+    P.SetBoneRotation('Bip01 L Calf');
+    P.SetBoneRotation('Bip01 R Calf');
+    P.SetBoneRotation('Bip01 L Foot');
+    P.SetBoneRotation('Bip01 R Foot');
+    
+    bAttachedDriver=false;
```
Comment (GreatEmerald) [inline Classes/UT3HellbenderRearGunPawn.uc]: Remove these as well, this variable is unused.

File: Classes/UT3HellbenderRearGunPawn.uc (hunk 3 above)
Comment (HellDragon-HK) [reply Classes/UT3HellbenderRearGunPawn.uc]: Doesn't work, it won't compile that way

File: Classes/UT3HellbenderRearGunPawn.uc (hunk 2 above)
Comment (HellDragon-HK) [reply Classes/UT3HellbenderRearGunPawn.uc]: C:\Unreal Anthology\UT2004\UT3Vehicles\Classes\UT3HellbenderRearGunPawn.uc(48) : Error, 'Var' is not allowed here

Comment (GreatEmerald) [reply Classes/UT3HellbenderRearGunPawn.uc]: Make them `local` if you want to go that way.

File: Classes/UT3HellbenderRearGunPawn.uc (hunk 3 above)
Comment (GreatEmerald) [reply Classes/UT3HellbenderRearGunPawn.uc]: It will as long as it is in the defaultproperties rotator form of NeckDrive=(Yaw=4000) etc.

**Comment by GreatEmerald** [comment]
OK, thanks.

**Pull request merged**